00-about-thursdays-at-ada/about-thursdays-at-ada.md

```diff
@@ -3,15 +3,15 @@
 In Computer Science fundamentals students learn the following concepts and data structures:
 
 
-- [Algorithmic Approaches](../01-algorithms/01-algorithms.md) such as Divide & Conquer, Dynamic 
+- [Algorithmic Approaches](../01-algorithms/01-algorithms.md)
 - [Linked Lists](../02-linked-lists/01-linked-lists.md)
 - [Binary Search Trees](../03-Binary-Search-Trees/01-Ordered-Collections-Of-Data.md)
 - [Graphs: BFS & DFS](../04-graphs/01-graphs.md)
-- [Graphs: Dijkstra](../05-dijkstras/01-graphs-review.md)Programming & Greedy Algorithms
+- [Graphs: Dijkstra](../05-dijkstras/01-graphs-review.md)
 
 ### Reverse Classroom
 
-CS Fundamentals is taught in a [flipped-classroom](https://omerad.msu.edu/index.php?option=com_content&view=article&id=162:what-why-and-how-to-implement-a-flipped-classroom-model&catid=27:teaching) style. Students are assigned a week prior to review a markdown lesson with accompanying video lesson and slides. In class students are expected to ask questions, review concepts, and work on exercises in groups. Students are also given time to engage in a mock interview with a peer.
+CS Fundamentals is taught in a [flipped-classroom](https://omerad.msu.edu/index.php?option=com_content&view=article&id=162:what-why-and-how-to-implement-a-flipped-classroom-model&catid=27:teaching) style. Students are provided with a Learn topic to review two weeks prior to a live review session. In class students are expected to ask questions, review concepts, and work on exercises in groups. 
 
 ### CS Fun Recorded Sessions
```

Comment:
I think we'll need to update these links/create the Panopto folders for them. I added a task in Asana for us to track: https://app.asana.com/0/1201700438643002/1203988207485365/f

Reply:
Yeah I think right now everything is in a general folder. I was going to ask Sylvia if there's a standard organization method we should be using on Tuesday.